addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -352,6 +352,10 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
 #products_grid:not(.o_wsale_layout_list) {
     @include media-breakpoint-down(lg) {
         @include wsale-break-table();
+
+        td.oe_td_ppr_3 {
+            width: 100%;
+        }
```

Comment:
It would be better to move the fix  inside the `wsale-break-table()` mixin, this way we'll have everything it's related to "table structure deviation" in the same place 👍 

Please note that `wsale-break-table()` behaviour varies according to the `$-list` flag variable.
To ensure that the rule is triggered in grid mode only, you'll have to do something like:

```scss
 if(!$-list) {
     [data-ppr="3"] .oe_product {
         width: 100%;
     }
 }
```

Reply:
Done. Thank you :pray: